examples/Sinusoid/sample.jl

```diff
@@ -0,0 +1,214 @@
+# # [Sample](@id sinusoid-example)
+
+# In this example we have a model that produces a sinusoid
+# ``f(A, v) = A \sin(\phi + t) + v, \forall t \in [0,2\pi]``, with a random
+# phase ``\phi``. We want to quantify uncertainties on parameters ``A`` and ``v``, 
+# given noisy observations of the model output.
+# Previously, in the emulate step, we built an emulator to allow us to make quick and 
+# approximate model evaluations. This will be used in our Markov chain Monte Carlo 
+# to sample the posterior distribution.
+
+# First, we load the packages we need:
+using LinearAlgebra, Random
+
+using Distributions, Plots
+using JLD2
+
+using CalibrateEmulateSample
+using CalibrateEmulateSample.Emulators
+using CalibrateEmulateSample.MarkovChainMonteCarlo
+
+rng_seed = 43
+rng = Random.MersenneTwister(rng_seed)
+
+## We will start with the Gaussian process from the emulate.jl script (run that first)
+# Get Gaussian process emulator from file
+# Load the ensembles generated in the calibration step.
+example_directory = @__DIR__
+data_save_directory = joinpath(example_directory, "output")
+# Get observations, true parameters and observation noise
+obs_file = joinpath(data_save_directory, "observations.jld2")
+y_obs = load(obs_file)["y_obs"]
+theta_true = load(obs_file)["theta_true"]
+Γ = load(obs_file)["Γ"]
+# Get emulators and prior
+emulator_file = joinpath(data_save_directory, "emulators.jld2")
+emulator_gp = load(emulator_file)["emulator_gp"]
+prior = load(emulator_file)["prior"]
+
+init_sample = vec([1.0, 1.0])
```

Comment:
It's good practice to start the markov chain where the emulators are accurate - e.g. among the training points ( e.g. the unconstrained mean of the final ensemble is a good place). This will reduce spin-ups and help with step-size optimization